browser/resources/settings/brave_leo_assistant_page/model_config_ui.ts

```diff
@@ -123,6 +135,23 @@ export class ModelConfigUI extends ModelConfigUIBase {
     this.modelRequestName = e.target.value
   }
 
+  onContextSizeChange_(e: any) {
+    /**
+     * A valid context size is any positive number.
+     * Other values will simply be ignored.
+     */
+    const parsedContextSize = parseInt(e.target.value, 10)
+    if (isNaN(parsedContextSize) || parsedContextSize < 1) {
+      this.contextSize = null
```

Comment:
could we use the previous value instead?

Reply:
I wasn't quite sure what to do here, honestly. Feels more like a UX issue. Is it best to simply remove the user's invalid input? Or should we empty the field? Or should we insert Brave's own default context size (i.e., 4000) for custom models? Not sure.